src/sentry/incidents/tasks.py

```diff
@@ -31,107 +25,11 @@
 from sentry.snuba.models import QuerySubscription
 from sentry.snuba.query_subscriptions.consumer import register_subscriber
 from sentry.tasks.base import instrumented_task
-from sentry.users.models.user import User
-from sentry.users.services.user import RpcUser
-from sentry.users.services.user.service import user_service
 from sentry.utils import metrics
-from sentry.utils.email import MessageBuilder
-from sentry.utils.http import absolute_uri
 
 logger = logging.getLogger(__name__)
 
 
-@instrumented_task(
-    name="sentry.incidents.tasks.send_subscriber_notifications",
-    queue="incidents",
-    silo_mode=SiloMode.REGION,
-)
-def send_subscriber_notifications(activity_id: int) -> None:
```

Comment:
I confirmed via sendgrid that we haven't sent one of these in the last 30 days. There are no `IncidentActivity` rows with a `type` of `comment` since 2021, and since `user` isn't passed to `update_incident_status` nobody was getting subscribed so we never found a subscriber to execute the code on lines 83 and 84.